src/images/kafka.rs

```diff
@@ -0,0 +1,124 @@
+use crate::{core::WaitFor, Image};
+use std::{collections::HashMap, path::PathBuf};
+
+const CONTAINER_IDENTIFIER: &str = "confluentinc/cp-kafka";
+const DEFAULT_TAG: &str = "6.1.1";
+
+pub const KAFKA_PORT: u16 = 9093;
+const ZOOKEEPER_PORT: u16 = 2181;
+
+#[derive(Clone, Debug)]
+pub struct KafkaArgs {
+    pub bash_pathname: PathBuf,
+    pub options: Vec<String>,
+    pub commands: String,
+}
+
+impl Default for KafkaArgs {
+    fn default() -> Self {
+        let starter_commands = format!(
+            "echo 'clientPort={}' > zookeeper.properties; \
+echo 'dataDir=/var/lib/zookeeper/data' >> zookeeper.properties; \
+echo 'dataLogDir=/var/lib/zookeeper/log' >> zookeeper.properties; \
+zookeeper-server-start zookeeper.properties & \
+export KAFKA_ZOOKEEPER_CONNECT='localhost:{}'; \
+export KAFKA_ADVERTISED_LISTENERS='PLAINTEXT://localhost:{},BROKER://localhost:9092'; \
```

Comment:
Do these have to be part of this command string or can we specify them via the `env_args` infrastructure.

Reply:
Glad you asked. It turns out they don't, which is a pleasant surprise to me. I removed them from the command and moved them to `env_args` in 81be19a.